src/frontend/src/pages/CalendarPage/DesignReviewDetailPage/UserAvailabilitesView.tsx

```diff
@@ -95,9 +94,9 @@ const UserAvailabilites: React.FC<UserAvailabilitiesProps> = ({
             >
               Unavailable
             </Typography>
-            <Box sx={{ maxHeight: '350px', overflowY: 'auto' }}>
+            <Box sx={{ maxHeight: '350px', overflowY: 'scroll' }}>
               {currentUnavailableUsers.map((user) => (
-                <Typography style={{ textAlign: 'center', fontSize }}>{fullNamePipe(user)}</Typography>
+                <Typography style={{ textAlign: 'center', fontSize: '16px' }}>{fullNamePipe(user)}</Typography>
```

Comment:
can you change this to 1em or 1.2em, whatever makes sense. this way the font size changes based on the size of the screen

Reply:
for some reason i do not see the white border on my end, so im not sure what to do to fix that. this is a screenshot of what i see
<img width="1512" alt="Screenshot 2024-04-10 at 18 07 10" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/125512346/c31a09e2-3412-4c41-bece-0ed4820656dd">

Reply:
It went away for me too, it should be good